package.json

```diff
@@ -70,6 +70,7 @@
     "url": "https://github.com/lifinance/types"
   },
   "dependencies": {
+    "axios": "^1.4.0",
```

Comment:
Can this be a devDependency? So axios is not required when installing the types package?

Reply:
totally, nice catch